### DISCUSSION TITLE
PokéDex terminado

### DISCUSSION THREAD

**Discussion by rickygzz** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#555] [Show and tell]
Les comparto mi PokéDex

[Despliegue en https://rickygzz.github.io/04/index.html](https://rickygzz.github.io/04/index.html)

[Código en GitHub](https://github.com/rickygzz/rickygzz.github.io/tree/main/04)

Son 4 de la mañana, es mejorable pero doy por terminado este proyecto, así que a dormir que a las 9 hay que trabajar 🥱

Saludos colegas,

**Comment by JonathanJRodriguez** [comment]
Buenos días @rickygzz !

Me gustó bastante tu pokedex, no muy poblado de animaciones y con muchas funcionalidades.
Felicidades por el excelente trabajo.

Saludos!

**Reply by rickygzz** [reply]
¡ Muchas gracias por tus generosos comentarios !

Saludos y buenas vibras 🙌🏽

**Comment by stariluz** [comment]
Uffff te quedo muy padre la parte de las estadisticas, se ven geniales!

**Reply by rickygzz** [reply]
¡ Gracias hermano ! A la orden
